sparkmagic/sparkmagic/kernels/kernelmagics.py

```diff
@@ -104,7 +104,7 @@ def help(self, line, cell="", local_ns=None):
   <tr>
     <td>sql</td>
     <td>%%sql -o tables -q<br/>SHOW TABLES</td>
-    <td>Executes a SQL query against the sqlContext.
+    <td>Executes a SQL query against the HiveContext(sqlContext) or SparkSession(spark).
```

Comment:
sqlContext is not guaranteed to be of HiveContext type. It depends on the Livy setting. I guess you can say it runs against sqlContext or spark variable name.